### PR TITLE
Projects - Add list with link to repositories

### PR DESCRIPTION
**Pull request by jomey**
I went the simple route and only listed the projects with their links. All the other details are on each page.

### CODE REVIEW

**Comment by jomey** [comment]
The failure on OS X is due to installing `pytorch` in one of the tutorials. I accepted that as a 'fix later' item

File: book/projects/index.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ You can find our comprehensive guide for organizing projects before, during, and
 
 Here is our current list of project for our {{dates}} {{ hackweek }} hackweek:
 
-| Project Name (with link to GitHub repo) | Short Description | Project Lead(s) |
-|:--------|:--------|:-----|
-| Sample Title | Sample Description | Leader |
+| Project Name (with link to GitHub repo)                                             |
+|:------------------------------------------------------------------------------------|
+| (FMI project)[https://github.com/snowex-hackweek/NASA-UW-Hackweek-2024-FMI-Project] |
```
Comment (spestana) [inline book/projects/index.md:20]:
```suggestion
| [FMI project](https://github.com/snowex-hackweek/NASA-UW-Hackweek-2024-FMI-Project) |
```

File: book/projects/index.md [hunk 2]
```diff
@@ -15,6 +15,11 @@ You can find our comprehensive guide for organizing projects before, during, and
...
+| (Data Set Map Tool)[https://github.com/snowex-hackweek/SnowEx-Map-Tool]             |
```
Comment (spestana) [inline book/projects/index.md:21]:
```suggestion
| [Data Set Map Tool](https://github.com/snowex-hackweek/SnowEx-Map-Tool)             |
```

File: book/projects/index.md [hunk 3]
```diff
@@ -15,6 +15,11 @@ You can find our comprehensive guide for organizing projects before, during, and
...
+| (DB Re-Structure)[https://github.com/snowex-hackweek/db-re-structure]               |
```
Comment (spestana) [inline book/projects/index.md:22]:
```suggestion
| [DB Re-Structure](https://github.com/snowex-hackweek/db-re-structure)               |
```

File: book/projects/index.md [hunk 4]
```diff
@@ -15,6 +15,11 @@ You can find our comprehensive guide for organizing projects before, during, and
...
+| (MicroSnowEx)[https://github.com/snowex-hackweek/microsnowex2024]                   |
```
Comment (spestana) [inline book/projects/index.md:23]:
```suggestion
| [MicroSnowEx](https://github.com/snowex-hackweek/microsnowex2024)                   |
```

File: book/projects/index.md [hunk 5]
```diff
@@ -15,6 +15,11 @@ You can find our comprehensive guide for organizing projects before, during, and
...
+| (SWESARR)[https://github.com/snowex-hackweek/swesarr]                               |
```
Comment (spestana) [inline book/projects/index.md:24]:
```suggestion
| [SWESARR](https://github.com/snowex-hackweek/swesarr)                               |
```

File: book/projects/index.md [hunk 6]
```diff
@@ -15,6 +15,11 @@ You can find our comprehensive guide for organizing projects before, during, and
...
+| (Snowmelt Timing)[https://github.com/snowex-hackweek/snowmelt-timing-2024]          |
```
Comment (spestana) [inline book/projects/index.md:25]:
```suggestion
| [Snowmelt Timing](https://github.com/snowex-hackweek/snowmelt-timing-2024)          |
```

**Review by spestana — COMMENTED** [review]
link syntax 🙃

**Review by aaarendt — APPROVED** [review]
looks good now! Should we wait to add any other project teams?

**Comment by jomey** [comment]
> looks good now! Should we wait to add any other project teams?

I used the ones from this organization that had commit activity

**Comment by scottyhq** [comment]
> The build is currently failing because of issues pulling data for SNOTEL

An option is to change the execution setting at this point to 'auto'... although I don't think they're currently cached in CI...
 
https://github.com/snowex-hackweek/website-2024/blob/039c4b600c21d4afe435ffd55429a2d96c144cde/book/_config.yml#L34

**Comment by jomey** [comment]
> > The build is currently failing because of issues pulling data for SNOTEL
> 
> An option is to change the execution setting at this point to 'auto'... although I don't think they're currently cached in CI...
> 
> https://github.com/snowex-hackweek/website-2024/blob/039c4b600c21d4afe435ffd55429a2d96c144cde/book/_config.yml#L34

Yeah, we didn't cache it before and we would need at least one successful run

**Comment by jomey** [comment]
Closes #26

**Pull request merged**